msteams-platform/bots/how-to/conversations/conversation-messages.md

```diff
@@ -439,6 +364,88 @@ Form completion message appears in Adaptive Cards while sending a response to th
 
 For more information on cards and cards in bots, see [cards documentation](~/task-modules-and-cards/what-are-cards.md).
 
+## Add notifications to your message
+
+You can add notifications to your message using the `Notification.Alert` property. Notifications alert users about new tasks, mentions, and comments. These alerts are related to what users are working on or what they must look at by inserting a notice into their activity feed. For notifications to trigger from your bot message, set the `TeamsChannelData` objects `Notification.Alert` property to *true*. Whether or not a notification is raised depends on the individual user's Teams settings and you can't override these settings. The notification type is either a banner, or both a banner and an email.
```

Comment:
> Notifications alert users about new tasks, mentions, and comments

This reads as if you can only notify if you need to alert a user about new tasks, mentions or comments. Can I suggest something similar to: "Notifications alert users to an event in your application, like a new task, mention or comment"

> The notification type is either a banner, or both a banner and an email

Is this correct? I wasn't aware of notifications sparking emails directly (apart form the "here's what you missed emails" which aren't tied to the notification itself)

Reply:
Updated